cmd/webpkgserver/webpkgserver.example.toml

```diff
@@ -202,6 +202,85 @@
   # bytes for the OCSP response.
   #AllowTestCert = false
 
+# IMPORTANT NOTE: the support of the ACME protocol and automatic renewal of
+# certificates is currently in the EXPERIMENTAL stage.  Once we have more
+# experience with people using it out in the wild, we will gradually move it to
+# PRODUCTION mode.
+#
+# ACME is a protocol that allows for automatic renewal of certificates.
+# Web Packager HTTP server uses an ACME library https://github.com/go-acme/lego
+# to handle certificate renewal. Automatic certificate renewal is enabled
+# in Web Packager HTTP server via the 'autorenewcert' flag. Turning the flag on
+# will enable Web Packager to automatically request certificate renewals
+# whenever it has determined that the current certificate is expired or about to
+# expire.
+#
+# SXG.ACME only needs to be present in the toml file if 'autorenewcert' command
+# line flag was turned on. Note that a recommended best practice for setting up
+# the cert renewal that minimizes both cost and bombarding your Certificate
+# Authority with requests is that for a multi-instance setup of Web Packager,
+# only one instance is setup to do automatic cert renewals and the rest of the
+# instances will just be configured to reload the fresh certificate from disk
+# when their in-memory copies expire.  This also implies that the cert paths
+# configured above in 'PEMFile' will be located on a shared filesystem
+# accessible by all Web Packager HTTP server instances.
+#
+# For the full ACME spec, see:
+# 	https://tools.ietf.org/html/draft-ietf-acme-acme-02
+# 	https://ietf-wg-acme.github.io/acme/draft-ietf-acme-acme.html
+[SXG.ACME]
+  # This field indicates that this Web Packager should attempt to auto renew
+  # certificates using ACME.
+  #EnableCertAutoRenew=false
+
+  # This is the ACME discovery URL that is used for ACME http requests to the
+  # Certificate Authority that doles out the certificates. Currently, the only
+  # CA that supports automatic signed exchange cert renewals is DigiCert:
+  # https://docs.digicert.com/certificate-tools/acme-user-guide/acme-directory-urls-signed-http-exchange-certificates/
```

Comment:
Maybe mention "Required when ACME is enabled." (see: `Cert.PEMFile`)

Reply:
Done.

Reply:
I don't see "Required when ACME is enabled." or anything similar, even in the latest commit. I guess you pushed a wrong thing?

Reply:
That's weird, not sure what happened.  Fixed now.

Reply:
Sorry but I can't tell what is updated...

Reply:
I think what happened is that you suggested to add "Required when ACME is enabled" then you said I should not add it to the "Enable = false" field, so I removed it.  It was added to the other fields below though.

Reply:
Oops, sorry, I made this comment at a wrong place.